packages/atlas/src/config/channelFilter.ts

```diff
@@ -1,7 +1,24 @@
 import { ChannelWhereInput } from '@/api/queries'
 
+type ChannelFilter = string[]
+
+const channelFilterIds: ChannelFilter = []
+const avatarPhotoFilterIds: ChannelFilter = []
+const coverPhotoFilterIds: ChannelFilter = []
```

Comment:
Haven't noticed this before but we have those separate lists for filtered avatars, covers, medias and thumbnails. Let's use a single list instead that just contains banned asset IDs and pass that list as a filter to all NOT asset filters.

Maybe it would be easier if we actually had all filters in a single `contentFilter` file?